app/policies/request_for_comment_policy.rb

```diff
@@ -6,19 +6,19 @@ def create?
   end
 
   def show?
-    admin? || author? || rfc_visibility
+    admin? || author? || rfc_visibility || author_in_programming_group?
```

Comment:
I'd propose to switch the order here:

```suggestion
    admin? || author? || author_in_programming_group? || rfc_visibility
```